docs/getting-started/figma.md

```diff
@@ -99,6 +99,13 @@ Need to update your designs? Perhaps the best part, is that once you hook it up,
 </video>
 <div class="note">Shown above in the demo web app. Keep in mind that this works for iOS and Android codebases as well.</div>
 
+### Best practices
+When using Figma and Diez together there is an important best practices worth noting: **Create a separate Figma file for your design tokens**.
```

Comment:
```suggestion
When using Figma and Diez together there are important best practices worth noting: **Create a separate Figma file for your design tokens**.
```

Reply:
Oh this should be "practice" not "practices". WIll push the fix.